OpenSim/Moco/Components/DeGrooteFregly2016Muscle.h

```diff
@@ -261,10 +261,13 @@ class OSIMMOCO_API DeGrooteFregly2016Muscle : public Muscle {
 
     /// The residual (i.e. error) in the muscle-tendon equilibrium equation:
     ///         residual = tendonForce - fiberForce * cosPennationAngle
+    /// This is always computed using implicit form of the model since the
+    /// explicit form will produce a zero residual for any guess of normalized
+    /// tendon force.
```

Comment:
It's great to have this comment but consider being more specific. Add that the reason why the residual is zero in explicit form is that the explicit forms solves for the derivative of the normalized tendon force that makes the residual 0.

For this function what value is used for the derivative of the normalized tendon force?

Reply:
I made this comment more specific based on your suggestion. 

For this method, the derivative of normalized tendon force will be whatever the state variable value is.